feugy/melodie
# feat(core): extracts images from file tags when applicable

### What's in there?
Are included here:

- [x] feat(core): extracts images from file tags when applicable

### How to test?

Using mp3tags or EasyTag, set cover inside file idV3 tags.
Then removes (or rename) `cover.{ext}` file in the same folder.
Re-import the track, and see the new cover file being created from the tag.
